oadr2/event.py

```diff
@@ -456,7 +457,7 @@ def remove_events(self, evt_id_list):
         '''
         Remove a list of events from our internal member dictionary
 
-        event_id_list - List of Event IDs 
+        event_id_list - List of Event IDs
```

Comment:
Please fix formatting in a separate commit.

Reply:
Fixed